assets/src/blocks/event-date/block.json

```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "gatherpress/event-date",
+	"version": "0.1.2",
+	"title": "GatherPress Event Date",
```

Comment:
rename `Event Date` it's already in GatherPress category so name doesn't require GatherPress